alerts/lib/alerttask.py

```diff
@@ -546,3 +538,10 @@ def parse_json_alert_config(self, config_file):
                 logger.error("FAILED to open the configuration file\n")
 
         return json_obj
+
+    def generate_full_doc(self, alert_body, alert_es):
+        full_alert = {}
```

Comment:
nit:  We could write

```python
return {
    '_id': alert_es['_id'],
    '_index': alert_es['index'],
    '_source': alert_body
}
```

Reply:
Agreed, that looks cleaner, added in https://github.com/mozilla/MozDef/pull/1528/commits/b180218dd782c53252118da2474e5a5fc18b46e2